Update docs considering changes on Rocky 24R2 (#63)
ansys/pyrocky: README.rst

|pyansys| |MIT| |python| |pypi| |codecov| |MIT| |black| |pre-commit|

> PyRocky is a Python client library to create, run and post-processing

Reviewer: ```suggestion
PyRocky is a Python client library to create, run, and postprocess
```
Oxford comma and simple verb form. (An Oxford comma refers to always putting a comma before an "and" or an "or" in a list of three or more items.

- `Getting started <https://rocky.docs.pyansys.com/version/stable/getting_started/index.html>`_: Learn
  how to install PyRocky in user mode and then launch it.
- `User guide <https://rocky.docs.pyansys.com/version/stable/user_guide/index.html>`_: Understand how to

>   use the Rocky PrePost Scripting with PyRocky.

Reviewer: ```suggestion
  use Rocky PrePost Scripting with PyRocky.
```
Is "Scripting" capitalized because it is part of a proper noun? If not, use a lower case "s" instead.

Author: _PrePost Scripting_ is the name of the feature in which you can automate Rocky using scripts.

> - `API reference <https://rocky.docs.pyansys.com/version/stable/api/index.html>`_: Understand PyRocky API
>   endpoints, their capabilities, and how to interact with them programmatically.

Reviewer: ```suggestion
- `API reference <https://rocky.docs.pyansys.com/version/stable/api/index.html>`_: Understand how to use
  Python to programmatically interact with PyRocky..
```
The suggested change is due to a new rephrasing as some developers have pointed out the "endpoint" isn't a term used by Python.

Author: I'll try to improve that in a following PR.
